.env.default

```diff
@@ -1,11 +1,17 @@
-DB_USER=service_stac
-DB_PW=service_stac
-DB_NAME=service_stac
-DB_NAME_TEST=test_service_stac
+DB_USER=postgres
+DB_PW=postgres
+DB_NAME=service_stac_local
```

Comment:
I would also change the default names in the docu for local DB setting, this make then easier for quickly setting up local db if needed.

Reply:
I changed it, although the quick setup is using docker-compose, it you really need to create a db in a PG running directly on the host then it's most likely cause you want to test stuff and probably need a different name anyway.